cmd/zed/permission.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	api "github.com/authzed/authzed-go/arrakisapi/api"
+	"github.com/cockroachdb/cockroach/pkg/util/treeprinter"
+	"github.com/jzelinskie/cobrautil"
+	"github.com/jzelinskie/stringz"
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+
+	"github.com/authzed/zed/internal/printers"
+)
+
+var permissionCmd = &cobra.Command{
+	Use:   "permission <subcommand>",
+	Short: "perform queries on the permissions in a permission system",
+}
+
+var checkCmd = &cobra.Command{
+	Use:               "check <user:id> <permission> <object:id>",
+	Short:             "check that a permission exists between a user and an object",
+	Args:              cobra.ExactArgs(3),
+	PersistentPreRunE: cobrautil.SyncViperPreRunE("ZED"),
+	RunE:              checkCmdFunc,
+}
+
+var expandCmd = &cobra.Command{
+	Use:               "expand <permission> <object:id>",
+	Short:             "expand a relation on an object",
+	Args:              cobra.ExactArgs(2),
+	PersistentPreRunE: cobrautil.SyncViperPreRunE("ZED"),
+	RunE:              expandCmdFunc,
+}
+
+func checkCmdFunc(cmd *cobra.Command, args []string) error {
+	var userNS, userID string
+	err := stringz.SplitExact(args[0], ":", &userNS, &userID)
```

Comment:
This needs to support users/subjects with right hand side relations optionally. If the relation is not specified, then and only then should it default to `...`